var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -62,7 +79,12 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
         "~hiop~ipopt+python @:1.5.0",
         msg="ExaGO Python wrapper requires at least one solver enabled.",
     )
-
+    conflicts(
+        "+hiop~mpi ^hiop@1.0.0:~mpi",
+        when="@1.5.1:1.6.1",
+        msg="#18 - builds with hiop and without MPI cause compile time errors",
+    )
```

Comment:
This may well extend to earlier HiOp versions, but this is best we know of for now